src/django/api/processing.py

```diff
@@ -84,11 +84,12 @@ def parse_xlsx(file, request):
                     if cell.row != 1:
                         cell.value = format_percent(cell.value)
 
-        header = ','.join([cell.value for cell in ws[1]])
+        header = ','.join([format_cell_value(cell.value) for cell in ws[1]])
 
         rows = ['"{}"'.format(
             '","'.join([format_cell_value(cell.value) for cell in ws[idx]]))
-                for idx in range(2, ws.max_row + 1)]
+                for idx in range(2, ws.max_row + 1)
+                if any(cell.value is not None for cell in ws[idx])]
```

Comment:
I was initially torn on this change because dropping rows will lead to there being a misalignment between the row index and the row number in the source worksheet. However, since blank rows will always just fail and a lot of them will slow down the current geocoding process since we  start up a batch job for every row regardless of status, I came around to the idea of discarding them.

Reply:
@jwalgran As an alternative approach, we could calculate the last row with any values, then stop processing after that row. That would ensure that the actual facility rows had the correct row index, but prevent us from booting up large numbers of rows from the end of the list. We would still start the batch job for some empty rows, if they were in the middle of the list, but I think that would be relatively infrequent.